### ISSUE TITLE
Do not change loglevel when changing filter level

### ISSUE THREAD

**Issue by Greek64** [nextcloud/logreader#68]
**Enhancement**

I find it a big design flaw to have the loglevel filter setting of the logreader app also change the loglevel setting configuration itself.

If I want to temporarily change the filtering loglevel to "error", does not mean that I don't want to keep logging the previously configured loglevel.

I find that the logreader app should not touch the configured loglevel in config.php at all.
I this is not possible, at least add an extra setting to have the possibility to change them seperately.

(This odd behavior actually cause me quite some trouble, as I was trying to figure out why the configured loglevel was not staying in config.php)

**Operating system**: Docker Nextcloud
**PHP version:** 7.1.13
**Nextcloud version:** 12.0.5
**Logread version:** 2.0.0

**Comment by arnowelzel** [comment]
This is still true for Nextcloud 13.0.4 as well.

The reason is this code in the logreader app, `apps/logreader/lib/Controller/LogController.php`, line 185-197:

```
        public function setLevels($levels) {
                $intLevels = array_map('intval', str_split($levels));
                $minLevel = 4;
                foreach ($intLevels as $level => $log) {
                        if ($log) {
                                $minLevel = $level;
                                break;
                        }
                }
                $this->config->setSystemValue('loglevel', $minLevel);
                $this->config->setAppValue('logreader', 'levels', $levels);
                return $minLevel;
        }
```

Removing the line `$this->config->setSystemValue('loglevel', $minLevel);` should solve this. Maybe I'll create a pull request later, when I have more time for testing.

**Comment by MorrisJobke** [comment]
Fixed by #72
